modules/apps/object/object-service/src/main/java/com/liferay/object/internal/bulk/selection/DefaultPermissionObjectBulkSelectionAction.java

```diff
@@ -145,6 +193,32 @@ private long _getObjectDefinitionId(long companyId) throws PortalException {
 		return objectDefinition.getObjectDefinitionId();
 	}
 
+	private JSONObject _mergeDefaultPermissionsForAssetType(
```

Comment:
what about _putRoleDefaultPermissions ? ( I would like to avoid the name assetType as probably in this class is never used)